docs/configuration/agent.md

```diff
@@ -1854,5 +1854,33 @@ quotas for container sandbox directories. Valid project IDs range from
 1 to max(uint32). (default `[5000-10000]`)
 </td>
 </tr>
+</table>
+
+## Systemd Integration
```

Comment:
Hmm,  I think this file may not be the proper place to add this section since this file is all about Mesos agent flags but nothing else. Maybe we should add this section into https://github.com/apache/mesos/blob/master/docs/mesos-containerizer.md?

Reply:
Thats a good point. Actually I'm not 100% happy with both. It have to be where it's more visible if the user have a problem and start to read the docs. I think, the user will have a look in the docs of the agents first. :man_shrugging:

Reply:
I see your point. But `agent.md` is only for documenting agent flags, and `Delegate=yes` is not an agent flags.

Reply:
ok, then I move it to containerizer.md. :-)

Reply:
Thanks, after you move it to containerizer.md, please mark this thread as resolved.